skiko/src/awtMain/kotlin/org/jetbrains/skiko/Actuals.awt.kt

```diff
@@ -41,7 +43,18 @@ internal actual fun makeDefaultRenderFactory(): RenderFactory =
     }
 
 internal actual fun URIHandler_openUri(uri: String) {
-    Desktop.getDesktop().browse(URI(uri))
+    val desktop = Desktop.getDesktop()
+    if (desktop.isSupported(Desktop.Action.BROWSE)) {
+        desktop.browse(URI(uri))
+    } else when (hostOs) {
+        OS.Linux -> {
+            URI(uri) // Validate URI
```

Comment:
This seems redundant now.

Reply:
This is to keep the behavior consistent with other platforms using `Desktop.browse()`, where `URI()` throws `URISyntaxException`. If a developer relies on catching `URISyntaxException`, then that will break on Linux, but with this it won't.

Reply:
No, it sounds reasonable, but please add a short comment explaining this reasoning.

Reply:
Added comment